### PR TITLE
Document editor

### PR DESCRIPTION
**Pull request by garytryan**
# **Rendering**

* [ ] Components render as designed - check storybook
* [ ] Storybook stories display all states

# **Behaviour**

* [ ] Components behave as designed - check storybook

# **Code Style**

* [ ] The file has less than 200 line.
* [ ] All functions are less than 25 lines long, and do **one** thing.
* [ ] Variable names are descriptive (says what it is).
* [ ] Components are only concerned with presentation.
  * methods - do not transform data
  * computed - only transforms data for presentation purposes, and logic is simple and concise
  * data - only contains properties directly related to presentation logic
* [ ] Tipe components are prefixed 'Tipe', exp 'TipeButton'
* [ ] Tipe components are refrenced in <templates> with kebab-case, exp 'tipe-button'
* [ ] Component name is CamelCase, exp { name: 'TipeButton' }
* [ ] Components have `data-tipe-ui="$options.name"` on the root element.

# Tests

**<template>**

* [ ] tests `data-tipe-ui` on root is as expected
* [ ] snapshot test - should render to string

**:props**

* [ ] all props are tested

**@events**

* [ ] all events are tested

# README

**:props**

* [ ] all props are clearly described and easy to understand

**@events**

* [ ] all events are clearly described and easy to understand

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,17 +29,16 @@
     "@tipe/tipe-interfaces": "^1.0.4-0",
     "autosize": "^4.0.2",
     "chokidar": "^2.0.4",
-    "moment": "^2.22.2",
```
Comment (gdi2290) [inline package.json]: we can compute this on the server with graphql argument specifying the timezone etc is we didn't want to include moment on the client

Comment (garytryan) [reply package.json]: nice idea.

It's possible to do it on the client, just not the way I was doing it as it makes the tests non-deterministic.

**Comment by gdi2290** [comment]
LGTM

**Pull request merged**